src/pages/Search/Search.module.css

```diff
@@ -18,14 +18,77 @@
 	margin:5%;
 	display: inline;
  }
+ .background{
+	background-image: url('src/assets/images/background.jpg');
```

Comment:
Replace the relative path to this external weblink:
https://images.pexels.com/photos/54455/cook-food-kitchen-eat-54455.jpeg

_This change is for the possible file structure change after deployment_

Reply:
i adjusted the size of image and also rotated upside down. Can i do that with the external link.

Reply:
The external image size will be adjusted the same way as the local image, you do need to make some adjustments to rotate the image tho.